Herald/Herald/Sensor/Motion/InertiaSensor.swift

```diff
@@ -69,11 +44,36 @@ class ConcreteInertiaSensor : NSObject, InertiaSensor {
         logger.debug("start")
         motionManager.accelerometerUpdateInterval = updateInterval
         motionManager.stopAccelerometerUpdates()
-        motionManager.startAccelerometerUpdates(to: operationQueue, withHandler: accelerometerHandler)
+        motionManager.startAccelerometerUpdates(to: operationQueue, withHandler: handleAccelerometerUpdates)
```

Comment:
Does this approach work with iOS 9.3+ or only more recent versions?

Reply:
Functions are first-class data in Swift, so this _ought_ to work with all versions of Swift.